src/parsers/swagger/index.ts

```diff
@@ -1,10 +1,19 @@
 import * as yml from 'js-yaml'
 import * as converter from 'oas-raml-converter'
-import { parserUsage } from '../../utils'
+import { addPrefix, getOptionValue, isOptionSet, parserUsage, toCamel } from '../../utils'
 
+let isYAML: boolean = false
+let isPrefix: boolean = false
+let isCamelCase: boolean = false
+
+let prefix: string = ''
 export async function parse(content: string, options: any): Promise<AST> {
-    let isYAML = false
-    if (options.hasOwnProperty('y') || options.hasOwnProperty('yaml')) { isYAML = true }
+    if (isOptionSet(options, 'c', 'camel-case')) { isCamelCase = true }
+    if (isOptionSet(options, 'p', 'prefix-type-name')) {
+        prefix = String(getOptionValue(options, 'p', 'prefix-type-name'))
```

Comment:
Why are you using the `String()` constructor as a function here? At first glance it seems unnecessary but I may be missing something. Perhaps `getOptionValue(..)` should always return a string